test/smoke/testApps/OpenTelemetryMetric/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/OpenTelemetryMetricTest.java

```diff
@@ -109,8 +109,8 @@ private void validateHistogramMetric(String name) throws Exception {
     // validate custom dimension
     Map<String, String> properties = md.getProperties();
     int aggregationIntervalMs = Integer.parseInt(properties.get("_MS.AggregationIntervalMs"));
-    assertThat(aggregationIntervalMs, greaterThanOrEqualTo(30000));
-    assertThat(aggregationIntervalMs, lessThanOrEqualTo(90000));
+    assertThat(aggregationIntervalMs, greaterThanOrEqualTo(60000));
+    assertThat(aggregationIntervalMs, lessThanOrEqualTo(180005));
```

Comment:
this is a very big range, and makes me think that the `aggregationIntervalMs` is not the interval but the total time since the first interval?

Reply:
what do you suggest reverting it back to 1 min? I thought it's test delay.. i didn't think that way. i was able to test interval at the reader level for sec, or even 15 mins..

Reply:
or i can remove this assert statement for now..